libgnucash/app-utils/gnc-gsettings.cpp

```diff
@@ -279,18 +279,52 @@ gnc_gsettings_remove_any_cb_by_func (const gchar *schema,
 }
 
 
+static gboolean gnc_gsettings_enum_bool_mapping_get (GValue *value,
+                                                     GVariant *variant,
+                                                     gpointer user_data)
+{
+    g_value_set_boolean (value,
+        !g_strcmp0 ((const gchar *)user_data,
+            g_variant_get_string (variant, nullptr)));
+
+    return true;
+}
+
+static GVariant* gnc_gsettings_enum_bool_mapping_set (const GValue *value,
+                                                      const GVariantType *expected_type,
+                                                      gpointer user_data)
+{
+    if (g_value_get_boolean (value)) {
```

Comment:
This maybe me but could you place the if `{}` on separate lines here and the function below.

Reply:
Fixed